app/com/arpnetworking/database/h2/triggers/AlertsUpdateEtagTrigger.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2016 Groupon.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.arpnetworking.database.h2.triggers;
+
+import org.h2.api.Trigger;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+/**
+ * Trigger to update Etag after every insert, delete or update statement.
+ *
+ * @author Deepika Misra (deepika at groupon dot com)
+ */
+public class AlertsUpdateEtagTrigger implements Trigger {
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void init(final Connection conn,
+            final String schemaName,
+            final String triggerName,
+            final String tableName,
+            final boolean before,
+            final int type) throws SQLException {}
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void fire(final Connection conn, final Object[] oldRow, final Object[] newRow) throws SQLException {
+        try (PreparedStatement stmt = conn.prepareStatement("SELECT NEXTVAL('portal.alerts_etag_seq');")) {
```

Comment:
Could this be generalized to any sequence by accepting the sequence name in the constructor?